installer/vagrant/ubuntu.sh

```diff
@@ -1,5 +1,53 @@
 #!/bin/bash
 
+showhelp() {
+    echo "Usage: ubuntu.sh [options]
+    -h, --help, -?
+                Display usage information
+    --hostname=HOSTNAME
+                Appends the hostname to the guest's /etc/hosts
+                "
+    exit 0
+}
+
+# On some boxes, the hostname is not set correctly, causing RabbitMQ to fail to
+# start. This adds the hostname to /etc/hosts, which allows it to work
+# correctly.
+hostname=
+while :; do
+    case "$1" in
+        -h|--help|\?)
+            showhelp
+            ;;
+        --hostname|-n)
+            if [ "$2" ]; then
+                hostname=$2
+                shift 2
+                continue
+            else
+                echo 'ERROR: Must specify a non-empty "--hostname HOSTNAME" argument.' >&2
+                exit 1
+            fi
+            ;;
+        --hostname=?*)
+            hostname=${1#*=} # Delete everything up to = and assign the remainder
+            ;;
+        --hostname=)
+            echo 'ERROR: Must specify a non-empty "--hostname HOSTNAME" argument.' >&2
+            exit 1
+            ;;
+        *)
+            break
+    esac
+    shift
+done
+
+exists=$(grep "$hostname" /etc/hosts)
+if [ -n "$hostname" -a -z "$exists" ]; then
+    echo "Appending $hostname to /etc/hosts"
+    echo "$hostname" >> /etc/hosts
+fi
+
 # Install modern alsa module for snd-hda-intel
 # slightly modernized from  https://github.com/naomiaro/vagrant-alsa-audio
 # https://wiki.ubuntu.com/Audio/UpgradingAlsa/DKMS
```

Comment:
The audio parts below are most likely the only thing specific to the `ubuntu.sh` script and I don't believe they have worked recently.

I think we should just get rid of alsa support in vagrant boxes out of the box. If you need alsa support in a vagrant box you can install it after the fact and it should work fine.

We could also check to see if the audio-hda drivers included with the distros support virtualized alsa devices in virtualbox and libvirt nowadays.

Reply:
Yeah those fail to install - the URL returns 404. I'm happy to just drop it then

Reply:
It also means that the Debian and Ubuntu scripts can be merged together